src/constants/gl_markers.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     'square-open': '□',
     diamond: '◆',
     'diamond-open': '◇',
-    cross: '➕',
+    cross: '+',
```

Comment:
Thanks. Reverting back to the current symbol is the way to go here. 

That said, we could (and should I'd say) make `cross` thicker and add `cross-thin` in [`v2`](https://github.com/plotly/plotly.js/issues/420).